packages/odyssey-react-mui/src/labs/DataFilters.tsx

```diff
@@ -290,15 +290,16 @@ const DataFilters = ({
     setFilters(updatedFilters);
   }, [inputValues, filtersProp]);
 
-  const AutocompleteOuterContainer = styled.div`
-    display: flex;
-    gap: 2;
-    align-items: center;
-    alignitems: "flex-end";
-  `;
-  const AutocompleteInnerContainer = styled.div`
-    width: "100%";
-  `;
+  const AutocompleteOuterContainer = styled("div")({
+    display: "flex",
+    gap: "2",
+    alignItems: "flex-end",
+  });
+
+  const AutocompleteInnerContainer = styled("div")({
+    width: "100%",
+  });
```

Comment:
Are we purposefully removing the `alignItems: "flex-end";` from the previous changes for some reason?

Reply:
I didn't remove `flex-end` alignment.. I actually removed `center`. 
This:
```
align-items: center;
alignitems: "flex-end";
```
became this:
```
alignItems: "flex-end",
```
(The browser would have cascaded the styles and removed `align-items: center` anyways)